internal/cmd/tui.go

```diff
@@ -335,9 +335,9 @@ func (m tuiModel) View() string {
 			intro := block.Intro()
 			words := strings.Split(intro, " ")
 			// todo(sebastian): this likely only works well for English
-			max := 9 - len(strings.Split(relFilename, string(filepath.Separator)))
-			if len(words) > max {
-				intro = strings.Join(words[:max], " ") + "..."
+			maxComponents := 9 - len(strings.Split(relFilename, string(filepath.Separator)))
```

Comment:
Changed a variable name, because `gosec` complained that it was overwriting a global symbol. AFAIK from Go 1.23 there are `max` and `min` functions.